pyedflib/edfwriter.py

```diff
@@ -440,17 +445,17 @@ def setAdmincode(self, admincode):
         self.admincode = admincode
         self.update_header()
 
-    def setGender(self, gender):
+    def setSex(self, sex):
         """
-        Sets the gender.
+        Sets the sex.
```

Comment:
```suggestion
        Sets the sex. Due to the edf specifications, only binary assignment is possible.
```

Reply:
I could add that, but:
- that was already true before this PR,
-  I don't see the need to justify ourselves, the EDF library implements the EDF specs as a matter of course.

Reply:
true, but people do not read the specs, so I'd like to have it added